src/displayapp/DisplayApp.cpp

```diff
@@ -246,6 +246,10 @@ void DisplayApp::Refresh() {
         lcd.Sleep();
         PushMessageToSystemTask(Pinetime::System::Messages::OnDisplayTaskSleeping);
         state = States::Idle;
+        // Since the active screen is not really an app, go back to Clock.
```

Comment:
Might want to move this to before the message is pushed to the system task, as the system task will turn off the SPI bus and external flash when it receives this message. This may cause some problems loading fonts for the watchface when you call LoadScreen()

This may also cause the watchface to flash briefly on sleep entry though. To resolve this you'd want to somehow turn off the display backlight earlier

Reply:
fixed. I don't think it made much of a difference, but you're right - it seems like a better location.